democracy/migrations/0052_auto_20200401_1115.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 2.2.10 on 2020-04-01 11:15
+
+import democracy.enums
+import django.core.files.storage
+from django.db import migrations, models
+import enumfields.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('democracy', '0051_auto_20200225_1349'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='section',
+            name='commenting_map_tools',
+            field=enumfields.fields.EnumIntegerField(default=0, enum=democracy.enums.CommentingMapTools, verbose_name='commenting_map_tools'),
+        ),
+        migrations.AddField(
+            model_name='sectioncomment',
+            name='commenting_map_tools',
+            field=enumfields.fields.EnumIntegerField(default=0, enum=democracy.enums.CommentingMapTools, verbose_name='commenting_map_tools'),
+        ),
+        migrations.AddField(
+            model_name='sectioncomment',
+            name='map_comment_text',
+            field=models.CharField(blank=True, max_length=255, null=True, verbose_name='map_comment_text'),
+        ),
+        migrations.AlterField(
+            model_name='sectionfile',
+            name='file',
+            field=models.FileField(max_length=2048, storage=django.core.files.storage.FileSystemStorage(location='/home/hienous/kerrokantasi/protected_media'), upload_to='files/%Y/%m', verbose_name='file'),
```

Comment:
@SanttuA Here as well path contains developer's path.

Reply:
Same as https://github.com/City-of-Turku/kerrokantasi/pull/15#discussion_r470429597